tests/infrastructure/instance_types/test_cpu_memory_hotplug_instancetype.py

```diff
@@ -112,22 +120,26 @@ def test_hotplug_cpu_instance_type(self, instance_type_hotplug_vm, hotplugged_si
     def test_migrate_snapshot_hotplugged_vm(self, hotplug_vm_snapshot_instance_type, instance_type_hotplug_vm):
         migrate_vm_and_verify(vm=instance_type_hotplug_vm, check_ssh_connectivity=True)
 
-    @pytest.mark.dependency(depends=[f"{TESTS_CLASS_NAME}::hotplug_cpu_instance_type"])
-    @pytest.mark.polarion("CNV-11403")
-    def test_restart_hotplugged_vm(self, instance_type_hotplug_vm):
-        restart_vm_wait_for_running_vm(vm=instance_type_hotplug_vm)
-
-    @pytest.mark.dependency(depends=[f"{TESTS_CLASS_NAME}::hotplug_cpu_instance_type"])
+    @pytest.mark.dependency(
+        name=f"{TESTS_CLASS_NAME}::decrease_cpu_value", depends=[f"{TESTS_CLASS_NAME}::hotplug_cpu_instance_type"]
+    )
     @pytest.mark.polarion("CNV-11404")
-    def test_decrease_cpu_value(self, instance_type_hotplug_vm):
-        hotplug_instance_type_vm(vm=instance_type_hotplug_vm, sockets=FOUR_CPU_SOCKETS)
-        assert_restart_required_codition(
+    def test_decrease_cpu_value(self, instance_type_hotplug_vm, four_sockets_instance_type):
```

Comment:
if we decrease the cpu that comes as part of unplugging and that is not supported. might need some changes when we have https://issues.redhat.com/browse/CNV-61104 fixed